lib/tape/ansible_runner.rb

```diff
@@ -41,7 +41,7 @@ class AnsibleRunner < ExecutionModule
          "Checks out app code, installs dependencies and restarts unicorns for "\
          "both FE and BE code."
   action :everything,
-         proc { ansible if valid_preconfigs },
+         proc { (valid_preconfigs)? ansible : puts("Not a Rails or JS app") },
```

Comment:
Don't use parentheses around a method call.<br>Surrounding space missing for operator `?`.